docs/personalization/filters.md

```diff
@@ -22,6 +22,20 @@ User profile-based filters are applicable in both publishing and eCommerce use c
 |Do not recommend items the user already consumed|The Personalization service stores the CONSUME events of every user for one year. When you activate this filter, the user will not be recommended the consumed content again.|
 |Max. repeated shows of identical recommendations per session|When you activate this filter and set a value, after a content item/product is recommended a certain number of times during the current user session, it is removed from all recommendation lists.|
 
+### Exclusions
+
+With Exclusion features you can exclude categories from the recommendation response by providing fixed category paths or by item category paths.
+
+!!! tip
+
+    You can use both options at the same time.
+
+- Exclude category of the context item - excludes items in the recommendation response from specified categories in the recommendation request. Select this checkbox if you want do not want to recommend a client items from the category of the currently rendered item. For example, 
```

Comment:
I think that it coulde be something like:  `excludes items in the recommendation response from the same categories as context item`
too many: 'want'